cmd/snap/cmd_run_test.go

```diff
@@ -98,7 +98,7 @@ func (s *SnapSuite) TestSnapRunAppIntegration(c *check.C) {
 	// mock installed snap
 	dirs.SetRootDir(c.MkDir())
 	defer func() { dirs.SetRootDir("/") }()
-	defer mockSnapConfine()()
+	defer mockSnapConfine(dirs.DistroLibExecDir)()
```

Comment:
Pardon my ignorance, but it looks like the argument for `mockSnapConfine()` is always dirs.DistroLibExecDir - do we really need this new argument in this case?